### ISSUE TITLE
Puffin ship unable to complete mission - can't plot a course to Nenia

### ISSUE THREAD

**Issue by erikarn** [endless-sky/endless-sky#10245]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

This is with 0.10.6.

There isn't an obvious way to plot a course with the puffin to the Nenia system. All it has is a normal hyperspace drive.

### Steps to Reproduce

* Complete the free worlds story line through to the Pug hyperspace invasion and "fight"
* Go spleunking
* Wander into this new storyline
* .. and get the puffin ship, but no way to plot a course

### Expected Behavior

... there's no obvious way to plot a path to the desired system without a jump drive or some other drive I don't know about?

### Screenshots

<img width="639" alt="Screen Shot 2024-06-20 at 10 33 17 AM" src="https://github.com/endless-sky/endless-sky/assets/1156589/03de5011-7892-470b-90fc-af2f3a86e68b">

### Operating System

MacOS X 12.7.4

### Game Source

GitHub Releases

### Game Version

0.10.6

**Comment by Hecter94** [comment]
There are wormholes in Edusa, Segesta, and Aescolanus which you'll need to go through in order to get from Pantica to Nenia. 

![image](https://github.com/endless-sky/endless-sky/assets/15916854/42e8b104-c85a-4170-8288-d17ea38f5d94)

**Comment by erikarn** [comment]
Yeah, I discovered this afterwards. It wasn't immediately obvious I needed to explore some more.

Did I miss some suggestion to explore? I didn't even know what that quantum key store did until I found those wormholes. :)

**Comment by solardawning** [comment]
Look at the bright side, now you know how to go straight to meet the Remnant next time you start a new game, no Jump Drive needed. :)
